apps/web/src/components/CommunityInterestDialog/DevelopmentProgramInterestItem.tsx

```diff
@@ -0,0 +1,175 @@
+import { useIntl } from "react-intl";
+import CheckCircleIcon from "@heroicons/react/20/solid/CheckCircleIcon";
+import ExclamationCircleIcon from "@heroicons/react/20/solid/ExclamationCircleIcon";
+import PlayCircleIcon from "@heroicons/react/20/solid/PlayCircleIcon";
+import PauseCircleIcon from "@heroicons/react/20/solid/PauseCircleIcon";
+import XCircleIcon from "@heroicons/react/20/solid/XCircleIcon";
+
+import {
+  DevelopmentProgramParticipationStatus,
+  FragmentType,
+  getFragment,
+  graphql,
+  Maybe,
+} from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { IconType } from "@gc-digital-talent/ui";
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+
+interface StatusInfo {
+  Icon: IconType;
+  iconStyles: Record<string, string>;
+  message: string;
+}
+
+const useStatusInfo = (
+  status?: Maybe<DevelopmentProgramParticipationStatus>,
+  completionDate?: Maybe<string>,
+): StatusInfo => {
+  const intl = useIntl();
+
+  const defaultStatusInfo = {
+    Icon: ExclamationCircleIcon,
+    iconStyles: {
+      "data-h2-color": "base(error)",
+    },
+    message: intl.formatMessage(commonMessages.missingInformation),
+  };
+  if (!status || !completionDate) {
+    return defaultStatusInfo;
+  }
+
```

Comment:
Shouldn't all participation statuses other than COMPLETED have null completion dates? In which case they would always have missing information.

Reply:
Oh, yeah that totally makes sense 😆

Reply:
Good cathc, should be better after [improve missing info check](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12524/commits/05731ebfb6f09b2c8f21c7e699ff85fc44b81638)